build.yaml

```diff
@@ -18,6 +18,9 @@ jobs:
           - script: yarn copyrightheaders
             displayName: check copyrightheaders
 
+          - script: yarn generate-scss-typings
```

Comment:
Why is this needed as a separate build.yaml step? I'd expect unit tests to be using the identity-object-proxy thing and for everything else to be having this happen via a `yarn build` task, no?

Reply:
Correct! Will remove.

Reply:
I've re-added this as per what I discovered yesterday.

For PR clarity:
We assumed that using the identity-obj-proxy would ensure that typings would not be required. This is not true.

The identity-obj-proxy is for Jest to be able to consume packages that it would normally not be able to consume (in this case scss). However, this has no links to ts-jest, which we use to transform our TS code. As a result, when the typings do not exist for ts-jest to validate, it will complain about not being able to find the module.

The suggested solution to this that I've seen online is to use the isolatedModules flag on ts-jest but that is not ideal as it disables type-checking on test files for jest runs; not necessarily the worst thing for us, since our webpack/build process would type-check those files anyways, but still not ideal.